ui/app/js/controllers/user_edit_controller.js

```diff
@@ -145,6 +146,7 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
           permissions: {}
         };
         sweetAlert("Saved", "Permission added.", "success");
+        $scope.showRole = !$scope.showRole;
```

Comment:
What is happening here? What I see is that when a user gets the first permission then `showRole`becomes `false`, when they get the second then `showRole` becomes `true` etc so `showRole` keeps toggling as permission is added..